Modules/_statisticsmodule.c

```diff
@@ -0,0 +1,120 @@
+/* statistics accelerator C extensor: _statistics module. */
+
+#include "Python.h"
+#include "structmember.h"
+#include "clinic/_statisticsmodule.c.h"
+
+/*[clinic input]
+module _statistics
+
+[clinic start generated code]*/
+/*[clinic end generated code: output=da39a3ee5e6b4b0d input=864a6f59b76123b2]*/
+
+
+static PyMethodDef speedups_methods[] = {
+    _STATISTICS__NORMAL_DIST_INV_CDF_METHODDEF
+    {NULL, NULL, 0, NULL}
+};
+
+/*[clinic input]
+_statistics._normal_dist_inv_cdf -> double
+   p: 'd'
+   mu: 'd'
+   sigma: 'd'
+[clinic start generated code]*/
```

Comment:
Let's add a trailing `/` to mark all of this arguments as positional only.  There's no point for keyword arguments in an internal helper function.